INSTALLATION.md

```diff
@@ -158,6 +160,26 @@ For mongosh info see: https://docs.mongodb.com/mongodb-shell/
 ...
 
 ```
+**Note**: Windows user may proceed to next section of this documentation. In order to complete step 7 of process, please follow instructions outlined in this section which is universal for all operating systems.
+
+</br>
+
+### Windows Specific- Instructions to setup local instance of MongoDB
+
+**It is recommended to have a local instance of MongoDB database instead of a cloud-based one, as it enhances the development experience and provides a more streamlined experience.**
+
+1. Download the latest version of MongoDB Community Server, which includes MongoDB Compass, from the following link:[MongoDB Community Server]( https://www.mongodb.com/try/download/community)
+2. Separately download the MongoDB Shell from the tools section at the following link:[Mongo Shell](https://www.mongodb.com/try/download/shell)
+3. Extract the downloaded shell folder, locate the "mongosh" application, and paste it to the following location: `Program Files` -> `MongoDB` -> `bin`.
+    [Note: **You will find the mongosh application inside the `bin` folder**]
+4. Add the path of the location where you pasted the "mongosh" application to your system's environment variables.
+5. Create a folder named "data" in the C drive and within it create a new folder named "db".
+6. Open a terminal and run the "mongosh" command.
+7. In the `.env` file of talawa-api, add the first section of the connection string (mongodb://127.0.0.1:27017/) to the MONGO_DB_URL section.
```

Comment:
1. How can they know for sure that this is the URL to use? In the non Windows example, it is very clear.
1. MongoDB could be running on a different port in its configuration, for example.

Reply:
I am removing the specific url section actually it  does not matter they can even copy the whole connection string-url it will still work.

Reply:
Please add the URL. See my comment in the most recent review.